### ISSUE TITLE
[Feature Request] More fields for @ Mentions

### ISSUE THREAD

**Issue by mpearon** [frenya/vscode-coffeebreak#5]
Having the ability to add more fields to the @ Mentions entries and have them show on the hover card would be great.
I've not looked through the extension codebase to see how the hover card is populated, so I'm not sure how feasible this would be.
```json
"TUser": {
     "fullname": "Test User",
     "email": "tuser@domain.com",
     "company": "Contoso",
     "dept": "Project Management Office",
     "phone": "867-5309",
     "ext": "1234"
}
```
Something like that, maybe?

**Comment by frenya** [comment]
Hi, this should be fairly easy to implement. Will look into it next week :)

**Comment by frenya** [comment]
Implemented in version 0.1.15. Let me know if you have any ideas for further improvement :)
